package.json

```diff
@@ -62,7 +62,7 @@
     "@babel/code-frame": "^7.10.4",
     "@babel/runtime": "^7.12.5",
     "@types/aria-query": "^5.0.1",
-    "aria-query": "5.1.3",
+    "aria-query": "5.3.0",
```

Comment:
Why `aria-query` version hardly locked? As for not event `5.3.2` version cause package duplication in deps tree